boards/nxp/frdm_mcxc242/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
why add an empty file with only a license?

Reply:
Removed the file .